packages/vite/src/node/index.ts

```diff
@@ -60,6 +60,7 @@ export type {
   LogType,
   LoggerOptions,
 } from './logger'
+export type { BindShortcutsOptions, CLIShortcut } from './shortcuts'
```

Comment:
This should be removed with the internal change too.

Reply:
Ah this issue. Maybe we could refactor a bit to have `bindShortcuts()` do a `server._shortcutsBound = true` in the function itself? That way we can remove https://github.com/vitejs/vite/blob/6ae1d2d17ad57013647c13a08c2a0c4d0a28543f/packages/vite/src/node/server/index.ts#L448-L451 and the types so it doesn't get bundled by api extractor.

Otherwise the other option is to post process and strip it off like
https://github.com/vitejs/vite/blob/6ae1d2d17ad57013647c13a08c2a0c4d0a28543f/packages/vite/scripts/postPatchTypes.ts#L19-L20

The first one would be easier and also prevents someone accessing it.

Reply:
You can also hide it in a weak map against the server, no?

Reply:
Found something else, tell me what you think